testing/src/test/scala/org/http4s/Http4sSuite.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s
+
+import munit._
+import org.http4s.laws.discipline.ArbitraryInstances
+
+/** Common stack for http4s' munit based tests
+  */
+trait Http4sSuite
+    extends CatsEffectSuite
+    with syntax.AllSyntax
+    with ArbitraryInstances
```

Comment:
In Cats, it was discussed to not extend syntax traits or arbitraries in the base test suite, so as to make the tests look more like the kind of code it would be testing.  We will have a few more imports in our tests without these two lines, but I think it might be better, and this is our chance at a fresh start.  What do you think?

Reply:
I'll remove everything we don't need right away.